Packs/Lacework/Integrations/Lacework/Lacework.py

```diff
@@ -74,6 +74,13 @@ def format_compliance_data(compliance_data):
 
         compliance_data = compliance_data['data'][0]
 
+        # If the user wants to filter on a recommendation ID
+        if rec_id:
+            # Iterate through all recommendations, removing irrelevant ones
+            for recommendation in compliance_data["recommendations"][:]:
+                if rec_id != recommendation["REC_ID"]:
+                    compliance_data["recommendations"].remove(recommendation)
+
```

Comment:
not about it but it looks that `rec_id` can be easily converted to an array supporting parameter.
(not familiar with the use-case but looks like a quick win)

Reply:
Thanks for the suggestion, Eli!  The initial use case for this was to simply return one result for playbooks, but checking against a list of recommendations still allows for that use case, and makes the command more extensible.  I'll push a revision in just a few minutes.